In [2]:
import pandas as pd

In [3]:
rock_df = pd.read_csv("Mineral ores round the world.csv")

C:\Users\GIGABYTE\anaconda3\lib\site-packages\IPython\core\interactiveshell.py:3165: DtypeWarning: Columns (13,19) have mixed types.Specify dtype option on import or set low_memory=False.
  has_raised = await self.run_ast_nodes(code_ast.body, cell_name,


In [5]:
rock_df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 304632 entries, 0 to 304631
Data columns (total 22 columns):
 #   Column      Non-Null Count   Dtype  
---  ------      --------------   -----  
 0   site_name   304621 non-null  object 
 1   latitude    304613 non-null  float64
 2   longitude   304613 non-null  float64
 3   region      31179 non-null   object 
 4   country     304617 non-null  object 
 5   state       292148 non-null  object 
 6   county      252873 non-null  object 
 7   com_type    300815 non-null  object 
 8   commod1     283601 non-null  object 
 9   commod2     21632 non-null   object 
 10  commod3     63139 non-null   object 
 11  oper_type   304632 non-null  object 
 12  dep_type    37483 non-null   object 
 13  prod_size   108352 non-null  object 
 14  dev_stat    304632 non-null  object 
 15  ore         70558 non-null   object 
 16  gangue      33008 non-null   object 
 17  work_type   25600 non-null   object 
 18  names       65340 non-null   object 
 19  or

This dataset has 22 variables and 304632 total observation. Each observation contains site where the mineral was taken, latitude, longtitude, region it was from, country it was from, state if applicable, county if applicable, com type , commods, oper type, dep type, prod type, dev stat, type of ore, gangue, work type, names, ore control(?), hrock type and arock type

This dataset contains x columns and x rows. 